src/Sylius/Behat/Context/Setup/ThemeContext.php

```diff
@@ -51,24 +53,32 @@
     private $channelManager;
 
     /**
+     * @var string
+     */
+    private $cacheDir;
+
+    /**
      * @param SharedStorageInterface $sharedStorage
      * @param ThemeRepositoryInterface $themeRepository
      * @param ThemeFactoryInterface $themeFactory
      * @param ChannelRepositoryInterface $channelRepository
      * @param ObjectManager $channelManager
+     * @param ContainerInterface $container
      */
     public function __construct(
         SharedStorageInterface $sharedStorage,
         ThemeRepositoryInterface $themeRepository,
         ThemeFactoryInterface $themeFactory,
         ChannelRepositoryInterface $channelRepository,
-        ObjectManager $channelManager
+        ObjectManager $channelManager,
+        ContainerInterface $container
     ) {
         $this->sharedStorage = $sharedStorage;
         $this->themeRepository = $themeRepository;
         $this->themeFactory = $themeFactory;
         $this->channelRepository = $channelRepository;
         $this->channelManager = $channelManager;
+        $this->cacheDir = $container->getParameter('kernel.cache_dir');
```

Comment:
It's still a quite dirty solution, for now it uses the path defined in `config_test.yml` and we can't acces the temporary directory path there. I still seek for the solution that will allow us to persist themes configurations in test environment.